capitals.rb

```diff
@@ -1,152 +1,345 @@
+    capital: "Jackson",
+    tally: 0
+  },
+
+  {
     name: "Missouri",
-    capital: "Jefferson City"
-}, {
+    capital: "Jefferson City",
+    tally: 0
+  },
+
+  {
     name: "Montana",
-    capital: "Helena"
-}, {
+    capital: "Helena",
+    tally: 0
+  },
+
+  {
     name: "Nebraska",
-    capital: "Lincoln"
-}, {
+    capital: "Lincoln",
+    tally: 0
+  },
+
+  {
     name: "Nevada",
-    capital: "Carson City"
-}, {
+    capital: "Carson City",
+    tally: 0
+  },
+
+  {
     name: "New Hampshire",
-    capital: "Concord"
-}, {
+    capital: "Concord",
+    tally: 0
+  },
+
+  {
     name: "New Jersey",
-    capital: "Trenton"
-}, {
+    capital: "Trenton",
+    tally: 0
+  },
+
+  {
     name: "New Mexico",
-    capital: "Santa Fe"
-}, {
+    capital: "Santa Fe",
+    tally: 0
+  },
+
+  {
     name: "New York",
-    capital: "Albany"
-}, {
+    capital: "Albany",
+    tally: 0
+  },
+
+  {
     name: "North Carolina",
-    capital: "Raleigh"
-}, {
+    capital: "Raleigh",
+    tally: 0
+  },
+
+  {
     name: "North Dakota",
-    capital: "Bismarck"
-}, {
+    capital: "Bismarck",
+    tally: 0
+  },
+
+  {
     name: "Ohio",
-    capital: "Columbus"
-}, {
+    capital: "Columbus",
+    tally: 0
+  },
+
+  {
     name: "Oklahoma",
-    capital: "Oklahoma City"
-}, {
+    capital: "Oklahoma City",
+    tally: 0
+  },
+
+  {
     name: "Oregon",
-    capital: "Salem"
-}, {
+    capital: "Salem",
+    tally: 0
+  },
+
+  {
     name: "Pennsylvania",
-    capital: "Harrisburg"
-}, {
+    capital: "Harrisburg",
+    tally: 0
+  },
+
+  {
     name: "Rhode Island",
-    capital: "Providence"
-}, {
+    capital: "Providence",
+    tally: 0
+  },
+
+  {
     name: "South Carolina",
-    capital: "Columbia"
-}, {
+    capital: "Columbia",
+    tally: 0
+  },
+
+  {
     name: "South Dakota",
-    capital: "Pierre"
-}, {
+    capital: "Pierre",
+    tally: 0
+  },
+
+  {
     name: "Tennessee",
-    capital: "Nashville"
-}, {
+    capital: "Nashville",
+    tally: 0
+  },
+
+  {
     name: "Texas",
-    capital: "Austin"
-}, {
+    capital: "Austin",
+    tally: 0
+  },
+
+  {
     name: "Utah",
-    capital: "Salt Lake City"
-}, {
+    capital: "Salt Lake City",
+    tally: 0
+  },
+
+  {
     name: "Vermont",
-    capital: "Montpelier"
-}, {
+    capital: "Montpelier",
+    tally: 0
+  },
+
+  {
     name: "Virginia",
-    capital: "Richmond"
-}, {
+    capital: "Richmond",
+    tally: 0
+  },
+
+  {
     name: "Washington",
-    capital: "Olympia"
-}, {
+    capital: "Olympia",
+    tally: 0
+  },
+
+  {
     name: "West Virginia",
-    capital: "Charleston"
-}, {
+    capital: "Charleston",
+    tally: 0
+  },
+
+  {
     name: "Wisconsin",
-    capital: "Madison"
-}, {
+    capital: "Madison",
+    tally: 0
+  },
+
+  {
     name: "Wyoming",
-    capital: "Cheyenne"
-}]
+    capital: "Cheyenne",
+    tally: 0
+}].shuffle
+play = 0
+puts "Welcome! lets test your knowledge of state capitals. Would you like to play?"
+game = gets.chomp
```

Comment:
semantic_variable_police 🚔 
`game` could be more semantic-- `user_input` maybe?